### ISSUE TITLE
None of the lessons are working

### ISSUE THREAD

**Issue by alexbujenita** [learn-co/learn-browser#5]
Learn.co isn't aware of tests passing or failing.

**Comment by LiveGood** [comment]
Yes, I noticed the same. All the supposed labs on in Chrome browser are not working. On every lab it says that hundreds of people are "working on this lab", when in fact no one is capable of finishing it. Is this project left behind?
